docs/en/sql-reference/sql-functions/dict-functions/dict_mapping.md

```diff
@@ -0,0 +1,215 @@
+      +------+------------+-------+--------------+
+      |    1 | a1         |     1 |            1 |
+      |    4 | a3         |     1 |            3 |
+      |    2 | a1         |     1 |            1 |
+      |    3 | a3         |     1 |            3 |
+      +------+------------+-------+--------------+
+      4 rows in set (0.02 sec)
+      ```
+
+    The usage of `dict_mapping` in this example can accelerate [deduplication calculations and JOIN queries](../../../using_starrocks/query_acceleration_with_auto_increment.md). Compared to the previous solutions for building a global dictionary to accelerate precise deduplication, the  solution by using `dict_mapping` is more flexible and user-friendly. Because the mapping values are directly obtained from the dictionary table at the stage "loading mapping relationships between keys and values into the table". You do not need to write statements to join the dictionary table to obtain mapping values. Additionally, this solution supports various data loading methods.<!--For detailed usage, please refer to xxx.-->
+
+**Example 3: If the mapping column in the table is not configured as a generated column, you need to explicitly configure the `dict_mapping` function for the mapping column when loading data into the table, obtain the values mapped to the keys.**
+
+> **NOTICE**
+>
+> The difference between Example 3 and Example 2 is that when importing into the data table, you need to modify the import command to explicitly configure the `dict_mapping` expression for the mapping column.
+
+1. Create a table.
+
+    ```SQL
+    CREATE TABLE dest_table2 (
+        id BIGINT,
+        order_uuid STRING,
+        order_id_int BIGINT NULL,
+        batch int comment 'used to distinguish different batch loading'
+    )
+    DUPLICATE KEY (id, order_uuid, order_id_int)
+    DISTRIBUTED BY HASH(id);
+    ```
+
+2. When simulated data loads into this table, you obtain the mapped values from the dictionary table by configuring `dict_mapping`.
+
+    ```SQL
+    MySQL [test]> INSERT INTO dest_table2 VALUES (1, 'a1', dict_mapping('dict', 'a1'), 1);
+    Query OK, 1 row affected (0.35 sec)
+    {'label':'insert_19872ab6-8a96-11ee-b29c-00163e03897d', 'status':'VISIBLE', 'txnId':'42'}
+
+    MySQL [test]> SELECT * FROM dest_table2;
+    +------+------------+--------------+-------+
+    | id   | order_uuid | order_id_int | batch |
+    +------+------------+--------------+-------+
+    |    1 | a1         |            1 |     1 |
+    +------+------------+--------------+-------+
+    1 row in set (0.02 sec)
+    ```
+
+**Example 4: Enable strict mode**
+
+When strict mode is enabled and the value mapped to the key that doesn't exist in the dictionary table is queried , an error, instead of `NULL`, is returned . It makes sure that a data row's key is first loaded into the dictionary table and its mapped value (dictionary ID) is generated before that data row is loaded into the target table.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/07e0853b-14ac-4425-bf60-7e38a01aed31)
remove the space preceding the period